server/src/routes/api/v1/sitesRouter.js

```diff
@@ -1,26 +1,60 @@
 import express from "express";
+import uploadImage from "../../../services/uploadImage.js";
+import cleanUserInput from "../../../services/cleanUserInput.js";
 import { Site } from "../../../models/index.js";
+import { ValidationError } from "objection";
+import SiteSerializer from "../../../serializers/SiteSerializer.js"
 
 const sitesRouter = new express.Router();
 
 sitesRouter.get("/", async (req, res) => {
     try {
         const sites = await Site.query();
-        return res.status(200).json({ siteList: sites });
+        
+        const serializedSites = sites.map(site => {
+            return SiteSerializer.getSummary(site)
+        })
+        return res.status(200).json({ siteList: serializedSites });
     } catch (error) {
         return res.status(500).json({ errors: error });
     }
 });
 
+sitesRouter.post("/", uploadImage.single("image"), async (req, res) => {
+    try {
+        const { body } = req;
```

Comment:
here we can access `req.user` for the current user passport is keeping track of - the current user is not a responsibility for the front end